features/polyExplorer/src/components/embeddedSankey/embeddedSankey.jsx

```diff
@@ -5,9 +5,9 @@ import { PolyChart } from "@polypoly-eu/poly-look";
 const jurisdictionColors = {
     "EU-GDPR": "#60E6DE",
     "Five-Eyes": "#EC453D",
-    Russia: "#FE8988",
-    China: "#C5271E",
-    Sonstige: "#A9B6C6",
+    Russia: "#EC453D",
```

Comment:
Wait, all the same colors?
Also, shouldn't this be in the CSS file?

Reply:
d3 charts don't really work with css or it becomes quite the class nightmare. Arne told me to make them the same color

Reply:
Can't you work around that by defining CSS variables and use them in code? I've seen how to do that for instance here https://www.joshwcomeau.com/css/css-variables-for-react-devs/ Point is, double source of values can always cause trouble.